src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import { SyntheticEvent } from 'react';
+import { Tab, TabsProps } from '../../types/tab';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  const onTabClickHandle = (event: SyntheticEvent, tab: Tab) => {
```

Comment:
Consider renaming `onTabClickHandle` to `handleTabClick` for better readability and following the naming conventions for event handlers. 📚
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364